packages/numeric-input/src/__stories__/getKnobs.js

```diff
@@ -0,0 +1,43 @@
+import { action } from "@storybook/addon-actions";
+import { boolean, select } from "@storybook/addon-knobs/react";
+
+const knobGroupIds = {
+  basic: "Basic",
+  form: "Form Attributes"
+};
+
+const knobLabels = {
+  value: "Value",
+  step: "Step",
```

Comment:
`step` and `error` are not part of the `getKnobs` return  so they are not showing up on `storybook`.